doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,18 +154,159 @@ to keep your client decoupled from implementation details.
+
+### List entities [GET /v2/entities{?limit,offset,count,type,id,idPattern,q,geometry,coords,attrs}]
+
+Retrieves a list of entities which match criteria defined by the following parameters: `id`, `type`, `idPattern`, `q`,
+`geometry` and `coords` attribute (see below for a detailed description of these parameters). A given entity
+have to match all the criteria to be retrieved (i.e. criteria are combined in a logical AND way).
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">id</td><td>A comma separated list of elements.
+  Retrieve entities which ID match one of the elements in the list.</td>
+</tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">type</td><td>A comma separated list of elements.
+  Retrieve entities which type match one of the elements in the list.</td>
+</tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">idPattern</td><td>A correctly formated regular
+  expression. Retrieve entities which ID matches the regular expression.</td>
+</tr>
+<tr><td style="padding-right: 40px; width: 140px;">q</td><td>
+
+<p>A query expression, composed of a list of statements separated by <code>;</code>, i.e.</p>
+
+<p><code>
+q=statement;statements;statement
+</code></p>
+
+<p>Each statement express a matching condition. The result of the query is the match on each one of the statements, e.g. a
+AND logical condition among the different statements. There are two kind of statements: <em>unary statements</em> and
+<em>binary staments</em>.</p>
+
+<p>Binary statements are composed by an attribute name (e.g. "temperature", and operator and a value, which format
+depends on the operator), e.g.:</p>
+
+<p><code>
+temperature==50<br/>
+temperature<=20
+</code></p>
+
+<p>The list of operators (and the format of the values they use) is as follows:</p>
+
+<ul>
+  <li>Equal: <code>==</code>. Two possibilities:
+    <ul>
+      <li>The value is a single element, e.g. <code>temperature==40</code> or a list of comma-separated values,
+      e.g. <code>color==black,red</code>. It matches entities whose attribute value is one of the values (OR clause).
+      Eg. entities which <code>color</code> is set to <code>black</code> will match, while entities which
+      <code>color</code> is set to <code>white</code> will not match.</li>
+      <li>The value is a range, expressed with an starting value, and ending value and <code>..</code>. It match if
+      the value is in the range (range includes the starting and ending values). I can be only used with attributed
+      of numeric or date types. E.g. <code>temperature==10..20</code> matchs if the temperature is greater than
+      10 and lower than 20 (or equal to 10 or qual to 20).</li>
+    </ul>
+  </li>
+  <li>Unequal: <code>!=</code>. Two possibilities:
```

Comment:
We might be more concise here and describe this operator as the negation of the former?